### ISSUE TITLE
Anti-histone antibody positivity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6592]
**Preferred term label:** Anti-histone antibody positivity

**Synonyms**

**Definition (free text, please give PubMed ID)**
Anti-histone antibodies are found in the serum of up to 75–95% of people with drug-induced lupus and 75% of idiopathic SLE and less frequently in scleroderma, rheumatoid arthritis and undifferentiated connective tissue disease.

**Parent term (use hpo.jax.org/app)**
Antinuclear antibody positivity HP:0003493

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Systemic Lupus Erythematosus OMIM:152700
Drug-induced Lupus Erythematosus ORPHA:231111
Scleroderma ORPHA:801
Rheumatoid Arthritis OMIM:180300

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
